styles/main.css

```diff
@@ -413,22 +413,34 @@ footer ul li a:focus {
     margin-bottom: 16px;
 }
 
-@media (min-width: 444px) {
+@media (min-width: 1200px) {
```

Comment:
All this grid re-working will likely have consequences across the whole site. I don't think "fixing" it in this branch is really appropriate. Could you propose PRs with these changes, explaining why you're changing them, before this branch is merged?